docs/logo-use.md

```diff
@@ -0,0 +1,12 @@
+# Use of CDS Hooks Logos
+
+<center>
+<img src="/images/logo.png">
+<img width="60" src="/images/logo-square.png">
+</center>
+
+The CDS Hooks logos are owned by The Children's Medical Center Corporation and may be used in accordance with guidelines and policy at [https://smarthealthit.org/cds-hooks-logo](https://smarthealthit.org/cds-hooks-logo). These guidelines describe how CDS Hooks logos can be used in connection with:
```

Comment:
Note that this is a 404 today; SMART will be adding content similar to https://smarthealthit.org/logo/ (but tailored for CDS Hooks, and without the need for a distinct "works with" logo.)